documentation/book/proc-configuring-kafka-bridge-authentication-tls.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-authentication.adoc
+
+[id='proc-configuring-kafka-bridge-authentication-tls-{context}']
+= Configuring TLS client authentication in Kafka Bridge
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+* If they exist, the name of the `Secret` with the public and private keys used for TLS Client Authentication, and the keys under which they are stored in the `Secret`
+
+.Procedure
+
+. (Optional) If they do not already exist, prepare the keys used for authentication in a file and create the `Secret`.
```

Comment:
We should mention here that secrets created by the UO can be used out of the box.

Reply:
NOTE: Secrets created by the User Operator may be used.